src/app/components/onc-organization/onc-organization-view.jsx

```diff
@@ -49,22 +55,8 @@ function ChplOncOrganizationView(props) {
         className={classes.header}
       />
       <CardContent className={classes.content}>
-        <Typography variant="body1" gutterBottom>
-          <strong>Organization code</strong>
-          { organization.acbCode ?? organization.atlCode }
-        </Typography>
-        <Typography variant="body1" gutterBottom>
-          <strong>Retired</strong>
-          { organization.retired ? 'Yes' : 'No' }
-          { organization.retired
-            && (
-              <>
-                <strong>Retirement Date</strong>
-                { getDisplayDateFormat(organization.retirementDay) }
-              </>
-            )}
-        </Typography>
-        { organization.website
+        <Box className={classes.subContentBox}>
+          { organization.website
```

Comment:
If the website value doesn't exist, then the box that holds it shouldn't either. This line needs to be above line 58